### ISSUE TITLE
VoiceOver

### ISSUE THREAD

**Issue by aDENTinTIME** [AudioBooth/AudioBooth#294]
VoiceOver is a powerful feature that can aide those with visual impairments in navigating their device. I think properly supporting it in AudioBooth should be a priority. This issue is intended to be an umbrella under which any **bugs** or **enhancements** relating to VoiceOver can be gathered. Feel free to create a new issue here on GitHub and if it relates to VoiceOver I will add it as a sub-issue to this one.

**Comment by jeremygrenier** [comment]
Hello, VoiceOver improvements to the Settings screen are now in the latest beta, 1.11 (1782929780), if you'd like to try it.

**Comment by aDENTinTIME** [comment]
This is a definite improvement! I've kept the discussion in #295 as most of what you've addressed is reported there.
